.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: fastapi-router-controller
```

Comment:
Since there already is an action that does this think, could you remove it?

Reply:
Again i would rather see on every push if the tests are failing

Reply:
Yes, that's fine. We can have just one action that trigger tests both in push and PR